### ISSUE TITLE
Default is `undefined` when run it

### ISSUE THREAD

**Issue by MishaKav** [noxhsxrk/Jira-to-Git-Branch-Name#4]
1. When I ran it it wrote `undefined` in the message like in the screenshot.
![image](https://github.com/user-attachments/assets/282e3b90-3bdd-48da-8ae8-53ec460c959c)

2. Event if I try to set the `default` to `empty` string, it still writes `undefined`

3. It will be nice to see the other options to (default prefix) in the list of extension options, like other:
![image](https://github.com/user-attachments/assets/38ae8d70-0f9b-45ca-aa66-0723d2dc7e8b)

4. In my case I want to **copy** `git checkout -b xxx` and **not** evaluate the command, it will be nice if I could decide what to do in settings or similar.

**Comment by noxhsxrk** [comment]
Does the "jiraToGitBranchName.defaultPrefix" appear in the user settings.json?

My settings look normal

![image](https://github.com/user-attachments/assets/4f9ff851-8691-4a1f-8bd7-8e532bcab9b4)

**Comment by MishaKav** [comment]
Yes, it appears. But since i leave this empty, I see the `undefined`.
